packages/statusbar/src/statusbar.ts

```diff
@@ -112,6 +123,13 @@ export class StatusBar extends Widget implements IStatusBar {
     super.dispose();
   }
 
+  private WindowZoomedLevel = () => {
```

Comment:
Small wording change to fit with the naming convention

```suggestion
  private _isWindowZoomed = () => {
```